lib/logstash/inputs/stomp.rb

```diff
@@ -59,6 +75,10 @@ def register
       connect
       subscription_handler # is required for re-subscribing to the destination
     }
+
+    # Display configuration information
+    @logger.info("Reconnect is disabled, plugin will not reconnect on connection error") if !@reconnect
+    @logger.info("Debug is disabled, turn on debug for full exceptions") if !@debug
```

Comment:
Logger level is managed centrally in logstash, so specific debug config in plugin must be deprecated, so this line is not needed